xrpl4j-crypto-parent/xrpl4j-crypto-core/src/main/java/org/xrpl/xrpl4j/crypto/core/keys/Seed.java

```diff
@@ -100,12 +100,32 @@ public static Seed secp256k1SeedFromEntropy(final Entropy entropy) {
     return new Seed(UnsignedByteArray.of(Base58.decode(base58EncodedSeed)));
   }
 
+  /**
+   * Construct a {@link Seed} from the supplied {@code base58EncodedSecret}. Values for this function are most commonly
+   * found from an XRP Faucet, for example for the XRP devnet or testnet. On the xrpl.org documentation page, this value
+   * is often referred to as an account "secret", but it is actually just a base58-encoded string that contains an
+   * encoded 16-bytes of entropy, in addition to other binary padding and identification data.
+   *
+   * @param base58EncodedSecret A {@link Entropy} to generate a {@link Seed} from.
```

Comment:
I think this `@param` description is off

Reply:
Fixed. See https://github.com/XRPLF/xrpl4j/pull/213/files#diff-a28cd18969c4ffb7aadfeef4ef676525ee30001ca9a1854cacef28b8da8d5499R109